### PR TITLE
xmousebuttoncontrol: Update to version 2.19.2

### LINKED ISSUE #5112: XMouseButtonControl has a new version: 2.19.2
"url": "https://vps2.highrez.co.uk/downloads/XMouseButtonControl%202.19.2%20Portable.zip",
    "hash": "E8AE30AC6929A737685E16080B6B14CE65FC2C5B9F01709C3AE5EAF1F82E7D35",

### PR DESCRIPTION
**Pull request by earnestma**
- Closes #5112

### CODE REVIEW

**Comment by earnestma** [comment]
e9544b4 Chatty is something I am working on adding and forgot to move out of; it's not part of this PR though. sorry for the confusion.

File: bucket/xmousebuttoncontrol.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-    "version": "2.18.8",
+    "version": "2.19.2",
     "description": "Windows tool that allows re-configuration and expand the capabilities of mouse.",
     "homepage": "https://www.highrez.co.uk/downloads/xmousebuttoncontrol.htm",
-    "license": "Unknown",
-    "url": "https://vps2.highrez.co.uk/downloads/XMouseButtonControl%202.18.8%20Portable.zip",
-    "hash": "f8376bb51234a9c4bb4a5f17c944a6274d880582c5d1266f1ffe86848aca6156",
+    "license": "Freeware",
+    "url": "https://vps2.highrez.co.uk/downloads/XMouseButtonControl%202.19.2%20Portable.zip",
+    "hash": "E8AE30AC6929A737685E16080B6B14CE65FC2C5B9F01709C3AE5EAF1F82E7D35",
```
Comment (Ash258) [inline bucket/xmousebuttoncontrol.json:7]:
```suggestion
    "hash": "e8ae30ac6929a737685e16080b6b14ce65fc2c5b9f01709c3ae5eaf1f82e7d35",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
L34:

```diff
-        "regex": "Version ([\\d.]+)\\s+\\(\\d"
+        "regex": "Version ([\\d.]+)</"
```

**Comment by Ash258** [comment]
As in other PR. Next PR which will not be opened with write access to the collaborators will be immediately closed.

**Pull request merged**